app/controllers/organizations_controller.rb

```diff
@@ -1,15 +1,22 @@
 class OrganizationsController < AdminController
   include Pagination
+  skip_after_action :verify_authorized
 
   def index
     authorize(:dashboard, :show?)
+    current_admin.can?(:view_reports, :facility_group)
 
     @users_requesting_approval = policy_scope([:manage, :user, User])
       .requested_sync_approval
       .order(updated_at: :desc)
 
     @users_requesting_approval = paginate(@users_requesting_approval)
 
-    @organizations = policy_scope([:cohort_report, Organization]).order(:name)
+    @organizations = current_admin
```

Comment:
Our new scoping strategy is unidirectional, so we need to go from FacilityGroup to Organization manually.